docs/security_automation.md

```diff
@@ -0,0 +1,44 @@
+# Use Case: Enhance Security Posture through Automation
+
+Software development requires an ever-growing set of security practices to be implemented in order to produce software that can be trusted. Generating Software Bill of Materials (SBOMs), running vulnerability scans, static analysis, signing artifacts and more.
```

Comment:
I think we need to call out interoperability of security tooling. Since the main focus is interoperability, then we should definitely highlight that where we can.

While it does paint the picture, I think adding something like `CDEvents will provide a set of security based events to help tooling be interoperable and ensuring best security practices.`